client/components/item/AddItemGlobal.js

```diff
@@ -39,18 +39,27 @@ export const AddItemGlobal = ({
    * @return {void}
    */
   const handleSubmit = () => {
-    dispatch(
-      addItemsGlobal({
-        name,
-        weight,
-        quantity,
-        type: categoryType,
-        unit,
-      }),
-    );
+    console.log(isConnected, 'is connected');
```

Comment:
Ok this is great, but I imagine we will be repeating it a ton throughout app. As every async dispatch is going to need this check. 

Should we make a reusable component to check for isConnected, then do dispatch logic accordingly?